src/controllers/controllerengine.h

```diff
@@ -35,6 +35,15 @@ class ControllerEngineConnection {
     QScriptValue context;
 
     void executeCallback(double value) const;
+
+    // Required for QList::removeAll to work correctly in
+    // ControlObjectScript::disconnectScriptFunction
+    inline bool operator==(const ControllerEngineConnection& other) {
+        return id == other.id;
+    }
+    inline bool operator!=(const ControllerEngineConnection& other) {
```

Comment:
Implementing this was not necessary to fix the bug, but I don't want someone to get confused in the future in case this operator is needed and `==` is implemented but not `!=`.